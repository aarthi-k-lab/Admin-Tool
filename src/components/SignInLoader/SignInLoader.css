.sign-in-loader {
  align-items: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
}
