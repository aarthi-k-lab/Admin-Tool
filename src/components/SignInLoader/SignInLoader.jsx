import React from 'react';
import CircularProgress from '@material-ui/core/CircularProgress';
import AppContainer from '../AppContainer';
import Body from '../Body';
import Center from '../Center';

const SignInLoader = () => (
  <AppContainer>
    <Body>
      <Center>
        <div>
          <CircularProgress size={50} />
          <br />
          Signing In...
        </div>
      </Center>
    </Body>
  </AppContainer>
);

export default SignInLoader;
