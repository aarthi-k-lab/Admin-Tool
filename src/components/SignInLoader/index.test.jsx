import React from 'react';
import renderer from 'react-test-renderer';
import SignInLoader from '.';

describe('<SignInLoader />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<SignInLoader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
