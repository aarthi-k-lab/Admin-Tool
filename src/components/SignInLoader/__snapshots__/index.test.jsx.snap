// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignInLoader /> snapshot test 1`] = `
<div
  className="components-AppContainer-AppContainer__app"
>
  <div
    className="components-Body-Body__body-container"
  >
    <section
      className="components-Body-Body__body"
    >
      <div
        className="components-Center-Center__center--expand components-Center-Center__center"
      >
        <div
          className="components-AppCenterDisplay-AppCenterDisplay__app-center-display"
        >
          <div
            className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
            role="progressbar"
            style={
              Object {
                "height": 50,
                "width": 50,
              }
            }
          >
            <svg
              className="MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                cx={44}
                cy={44}
                fill="none"
                r={20.2}
                strokeWidth={3.6}
                style={Object {}}
              />
            </svg>
          </div>
          <br />
          Signing In...
        </div>
      </div>
    </section>
  </div>
  <footer
    className="components-Footer-Footer__footer"
  >
    Â© Nationstar Mortgage LLC, All rights reserved
  </footer>
</div>
`;
